categorize_notify/program.py

```diff
@@ -76,12 +118,15 @@ def _create_label(gmail, label_name: str) -> str:
         "messageListVisibility": "show",
         "name": label_name,
     }
-    created_label = gmail.labels().create(userId="me", body=label).execute()
-    print(f"Label created: {created_label['name']}")
-    return created_label["id"]
+    try:
+        created_label = gmail.labels().create(userId="me", body=label).execute()
+    except Exception as e:
+        print(f"Error creating label: {e}")
```

Comment:
I think this `try-except` block is unnecessary: now that exceptions dont' get swallowed by the runtime, we should keep raising this as an error, not just report it in the session log. Unless your think otherwise?

Reply:
You're right. I thought it was needed, but it was a different issue that I fixed. `label_name` was sometimes `None` and that was causing an error.